src/include/nfs/routines.rb

```diff
@@ -184,6 +190,45 @@ def legacy_entry?(entry)
       entry["vfstype"] == "nfs4" || NfsOptions.legacy?(entry["mntops"] || "")
     end
 
+    # Transforms a hash representing an NFS mount from the internal format used
+    # by yast-nfs-client to the TargetMap format used by the old yast-storage
+    #
+    # This is a direct translation of the old NfsClient4partClient#ToStorage
+    #
+    # @param entry [Hash] NFS mount in the internal format that uses keys as
+    #   "spec", "file", etc.
+    # @return [Hash] NFS mount in the TargetMap format that uses keys as
+    #   "device", "mount", etc.
+    def fstab_to_storage(entry)
+      ret = {}
+
+      if entry && entry != {}
```

Comment:
this is a bit inconsistent to below method which use early return.

Reply:
Well, it's basically moved code, so I tried to be conservative with the changes.

Reply:
ok